build/build_docker.ps1

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/pwsh
+
+# builds and pushes a docker file to docker hub
+# currently only builds the 'beta' tag, which is applied to whatever version
+# is supplied to this script.
+
+# get the current git commit
+$GIT_LONG_COMMIT = git rev-parse --long HEAD
+$GIT_SHORT_COMMIT = git rev-parse --short HEAD
+$SHORT_VERSION = docker run --rm relizaio/versioning -s YYYY.0M.0D.Calvermodifier -i beta
+$LONG_VERSION = docker run --rm relizaio/versioning -s YYYY.0M.0D.Calvermodifier+Metadata -i beta -m $GIT_SHORT_COMMIT
```

Comment:
document the options used in the format string inline - or at least link to the documentation

Also, should quote the format string

Reply:
the `beta` tag should not be included int the versions - it is not a true beta.